pyttb/sptensor.py

```diff
@@ -520,26 +664,29 @@ def double(self) -> np.ndarray:
 
     def elemfun(self, function_handle: Callable[[np.ndarray], np.ndarray]) -> sptensor:
         """
-        Manipulate the non-zero elements of a sparse tensor
+        Apply a function to the non-zero elements of a sparse tensor.
```

Comment:
If really motivated we have both non-zero and nonzero throughout the class. I think nonzero is more common.

Reply:
Agreed. WIll make the change.